### DISCUSSION TITLE
任务执行时间和本机时间不一致，有小伙伴碰到过吗？

### DISCUSSION THREAD

**Discussion by Timiyu** [whyour/qinglong#1644] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17613068/194443523-c506e841-04ba-4012-b9ef-c23a2dafafce.png)

**Comment by cc9014** [comment, marked as answer]
应该是时区的问题，先用 `date` 或 `date -R`查看是不是东八区。
